udata/commands/db.py

```diff
@@ -99,10 +99,11 @@ def execute_migration(plugin, filename, script, dryrun=False):
     return success
 
 
-def record_migration(plugin, filename, script):
+def record_migration(plugin, filename, script, **kwargs):
     '''Only record a migration without applying it'''
     db = get_db(DEFAULT_CONNECTION_NAME)
     db.eval(RECORD_WRAPPER, plugin, filename, script)
+    return True
```

Comment:
I think the error is already handled by MongoEngine so in case of misconfiguration you will have the information to correct it.
Do you see something more useful to do ?